src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Log4NetExtensions.cs

```diff
@@ -58,5 +59,54 @@ public static ILoggerFactory AddLog4Net(this ILoggerFactory factory)
             factory.AddLog4Net(DefaultLog4NetConfigFile);
             return factory;
         }
+
+#if !NETCOREAPP1_1        
```

Comment:
The problem here is !NETCOREAPP1_1 involves:

- NETCOREAPP2.0
- NET462
- NETSTANDARD2.0

Is this your intention? Or maybe you want to be only available on .NETCORE2.0? If so, maybe it is good to change the condition to #if NETCOREAPP2_0

Reply:
No, I want to support all except NETCOREAPP 1.1 because the ILoggingBuilder does not exists there.